components/movementsensor/dualgps/dualgps.go

```diff
@@ -263,8 +263,8 @@ func (dg *dualGPS) Orientation(ctx context.Context, extra map[string]interface{}
 	return spatialmath.NewZeroOrientation(), movementsensor.ErrMethodUnimplementedOrientation
 }
 
-func (dg *dualGPS) Accuracy(ctx context.Context, extra map[string]interface{}) (map[string]float32, error) {
-	return map[string]float32{}, movementsensor.ErrMethodUnimplementedAccuracy
+func (dg *dualGPS) Accuracy(ctx context.Context, extra map[string]interface{}) (*movementsensor.Accuracy, error) {
```

Comment:
I think the two places where we should revisit the returns are in the `dual_gps` and `merged` drivers, where we can bubble up the accuracies from the sensors we depend on.